csharp/Worker/Microsoft.Spark.CSharp/App.config

```diff
@@ -11,7 +11,9 @@
     <root>
       <level value="INFO" />
       <appender-ref ref="ConsoleAppender" />
+      <!--
       <appender-ref ref="LogFileAppender" />
+      -->
```

Comment:
I considered removing this line directly. 
However, IMO, if we comment it out instead of deleting it, it will be much easier for Mobius users to notice that there is an OOTB `LogAppender` which writes logs to files. If users use nuget package, there is no git history available.